manifests/profile_rams_full_stack.pp

```diff
@@ -1,12 +1,14 @@
-# helper profile that includes all stuff needed to setup a fully-functional
+# helper profile that includes all stuff needed to setup a fully-functional ubersystem server
 
 class uber::profile_rams_full_stack (
 
 ) {
+  include nginx
+
   require ::uber::firewall
   require ::uber::permissions
-  require ::uber::app
   require ::uber::db
+  require ::uber::app
```

Comment:
neat thing, we switch the order here so postgres finished before the app starts, which will kill some of our first-time-start spam about needing to retry connecting to the DB